mixer/pkg/attribute/responeFlagParser.go

```diff
@@ -0,0 +1,139 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package attribute
+
+import (
+	accessLogDataGRPC "github.com/envoyproxy/go-control-plane/envoy/data/accesslog/v2"
```

Comment:
super nit: just `alspb` or `accesslog` or `accesslogpb` would be fine here (the protocol is irrelevant).